playbooks/ansible-test-network-integration-base/templates/inventory-junos.j2

```diff
@@ -2,6 +2,7 @@
 #  DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
 #
 [junos]
+vqfx-18.1R3 ansible_host={{ hostvars['vqfx-18.1R3'].ansible_host }} ansible_user=zuul
```

Comment:
So, this won't work, as each nodeset doesn't know of the other. So we need to see if the host exists in hostvars, or do something like hostvars[appliance[0]]

To get the first host in the appliance group

Reply:
The think to remember, hostvars is passed from outer ansible (zuul-executor) into inner zuul